src/components/views/rooms/EventTile.js

```diff
@@ -389,49 +373,63 @@ module.exports = React.createClass({
             mx_EventTile_last: this.props.last,
             mx_EventTile_contextual: this.props.contextual,
             menu: this.state.menu,
-            mx_EventTile_verified: this.state.verified == true,
-            mx_EventTile_unverified: this.state.verified == false,
+            mx_EventTile_verified: this.state.verified === true,
+            mx_EventTile_unverified: this.state.verified === false,
         });
-        var timestamp = <a href={ "#/room/" + this.props.mxEvent.getRoomId() +"/"+ this.props.mxEvent.getId() }>
-                            <MessageTimestamp ts={this.props.mxEvent.getTs()} />
-                        </a>
 
-        var aux = null;
-        if (msgtype === 'm.image') aux = "sent an image";
-        else if (msgtype === 'm.video') aux = "sent a video";
-        else if (msgtype === 'm.file') aux = "uploaded a file";
+        // Removed for now, pending discussion on it's value and complexity
```

Comment:
I think it is very useful to be able to get full details of who has read the message and when.  @antikewl previously proposed a design for an extended message details view that might be triggered from the contextual menu (or in fact replace the contextual menu - turning it into a "show more info" button, which then revealed additional options and details for the message in a section below).  I do *not* propose we creep towards that design yet, but it serves to show that we really do want the option of being able to see precisely which users have read a message or not.

That said, I don't think that anyone has been relying particularly on the old implementation, so we could try shipping without it and see if anyone notices...